scripts/zones/La_Theine_Plateau/npcs/qm3.lua

```diff
@@ -23,13 +25,33 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
+    if
```

Comment:
We should just move this questline into interaction framework as it would not only futureproof this but make it much more resilient.

Reply:
aye that's in the PR todo. I will be doing it asap, but pushing everything else for now until I can get to it, allowing testers to get their hands on the quest sooner if necessary. (I've never worked with the framework and it's a little foreign to me, for the time being, until I have more time to delve into it)